src/cmap/wire_protocol/compression.ts

```diff
@@ -61,6 +64,15 @@ export function compress(
       }
       zlib.deflate(dataToBeCompressed, zlibOptions, callback as zlib.CompressCallback);
       break;
+    case 'zstd':
+      if ('kModuleError' in ZStandard) {
+        return callback(ZStandard['kModuleError']);
+      }
+      ZStandard.compress(dataToBeCompressed, self.options.zstdCompressionLevel).then(
```

Comment:
Should we pick a default level here in the driver? Also we may want to expose this as a config at the client level but I think we can file a follow up for that

Reply:
I appreciate the enthusiasm but we should not add an option to configure this at this time. There's system dependent levels that are supported server side but not always on the client side, I'd want us to confer with other drivers before shipping this knob.
Relevant thread: https://mongodb.slack.com/archives/C0UD1CGUF/p1652274501376419

However I do think it's good for us to pick 3 explicitly for now so we don't rely on an underlying default that could change (ideally not)

Reply:
I've removed configurability for this now and just made a constant that defaults it to 3.